vendor/assets/javascripts/leaflet_solr_heatmap.js

```diff
@@ -334,7 +334,9 @@ L.SolrHeatmap = L.GeoJSON.extend({
     }
 
     var bounds = this._map.getBounds();
-    return [bounds.getWest(), bounds.getSouth(), bounds.getEast(), bounds.getNorth()].join(',');
+    var wrappedSw = bounds.getSouthWest().wrap();
```

Comment:
No. that was a first pass, and hasn't been updated yet. It has diverged a bit, but tech debt would to bring upstream inline